app/namespace.go

```diff
@@ -1492,10 +1648,12 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 		},
 	}
 
+	export_s3_commands.Subcommands = append(export_s3_commands.Subcommands, export_general_commands...)
 	exportCommand.Subcommands = append(exportCommand.Subcommands, export_s3_commands)
 
 	// TODO: remove GCP sink feature flag check when out of private preview
```

Comment:
[Nit] let's update comment to reflect new nomenclature (e.g. "pre-release" instead of "private preview")